### PR TITLE
Add background setting for canvas nest

### PR DESCRIPTION
**Pull request by jinfang134**
for issues 想给博客添加canvas-nest.js的效果，但是不知道该怎么添加 #1325

添加了脚本canvas-nest.js的引用，并加了配置项。

### CODE REVIEW

**Comment by ivan-nginx** [comment]
@jinfang134 also, http://jinfang.oschina.io don't load in some countries: https://check-host.net/check-http?host=http://jinfang.oschina.io/
I can't see your blog.

File: _config.yml [hunk 1]
```diff
@@ -121,6 +121,14 @@ font:
...
     size:
 
+# --------------------------------------------------------------
+# background settings
+# --------------------------------------------------------------
+
+
+# see detail from https://github.com/hustcc/canvas-nest.js
+# issues 想给博客添加canvas-nest.js的效果，但是不知道该怎么添加 #1325
+#canvas_nest: true
```
Comment (AlynxZhou) [inline _config.yml]: I've done a better way to add canvas which, I think, is more NexT-liked. #1327

**Comment by jinfang134** [comment]
OK,I see.
Could you tell me where are you. I am in China and  it is so slow to visit the github.com , so I didn't use the github page for my blog.
----- 原始邮件 -----
发件人："Ivan.Nginx" <notifications@github.com>
收件人：iissnan/hexo-theme-next <hexo-theme-next@noreply.github.com>
抄送人：jinfang134 <jinfang134@sina.com>,  Mention <mention@noreply.github.com>
主题：Re: [iissnan/hexo-theme-next] Add background setting for canvas nest (#1326)
日期：2016年12月25日 17点07分

@jinfang134 also, http://jinfang.oschina.io don't load in some countries: https://check-host.net/check-http?host=http://jinfang.oschina.io/

I can't see your blog.

&mdash;

**Comment by ivan-nginx** [comment]
@jinfang134 this is e-mail or what?
From Russia. U may check your host by the link at the top.

**Comment by jinfang134** [comment]
thank you for you help,this is my first times to contribute to this 
project and also my first times to contribute to open source project.

I will improve it for the next imlement.

**Comment by ivan-nginx** [comment]
@jinfang134 ok, don't worry about it.
I think, u may close this pull because #1327 is ready and worked fine.
But, i also think, what u can resume your work with #1324 pull comments plugin.

**Pull request closed without merging**